L1Trigger/L1TTrackMatch/plugins/L1TkFastVertexProducer.cc

```diff
@@ -148,28 +137,28 @@ L1TkFastVertexProducer::~L1TkFastVertexProducer() {
 //
 
 // ------------ method called to produce the data  ------------
-void L1TkFastVertexProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup) {
+void L1TkFastVertexProducer::produce(edm::StreamID, edm::Event& iEvent, const edm::EventSetup& iSetup) const {
   using namespace edm;
 
-  std::unique_ptr<TkPrimaryVertexCollection> result(new TkPrimaryVertexCollection);
+  auto result = std::make_unique<TkPrimaryVertexCollection>();
 
   // Tracker Topology
   edm::ESHandle<TrackerTopology> tTopoHandle_;
   iSetup.get<TrackerTopologyRcd>().get(tTopoHandle_);
```

Comment:
Could we also get esConsumes used with the appropriate token?

Reply:
Sure, I'll do that now.